### PR TITLE
Fixed: Note resize

### PR DESCRIPTION
**Pull request by CasuaIAz**
I also increased minWidth a bit, because user can resize to much and the buttons can overlap with note names.

### CODE REVIEW

File: ToNote/Converters/BoolToResizeModeConverter.cs [hunk 1]
```diff
@@ -19,6 +19,25 @@ public object Convert(object value, Type targetType, object parameter, System.Gl
                 throw new ArgumentException("Wrong data type. Expected a boolean");
         }
 
+        public object ConvertBack(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            throw new NotImplementedException();
+        }
+    }
+    public class WidthConverter : IValueConverter
```
Comment (t-eliks) [inline ToNote/Converters/BoolToResizeModeConverter.cs]: This should be a separate file. Converters/WidthConverter.cs

File: ToNote/Converters/BoolToResizeModeConverter.cs [hunk 2]
```diff
@@ -19,6 +19,25 @@ public object Convert(object value, Type targetType, object parameter, System.Gl
...
+    {
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            double result = 0;
```
Comment (t-eliks) [inline ToNote/Converters/BoolToResizeModeConverter.cs]: `if (value == null) return 0;
return (double)value - 60;`

**Review by Kodnot — APPROVED** [review]
Good

**Comment by Kodnot** [comment]
[Fix resize](https://trello.com/c/tfcABre1/84-fix-resize)

**Pull request merged**